api/contrib/envoy/extensions/tracers/fluentd/v3alpha/fluentd.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+package envoy.extensions.tracers.fluentd.v3alpha;
+
+import "envoy/config/core/v3/backoff.proto";
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/wrappers.proto";
+
+import "udpa/annotations/migrate.proto";
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.tracers.fluentd.v3alpha";
+option java_outer_classname = "FluentdProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/contrib/envoy/extensions/tracers/fluentd/v3alpha";
+option (udpa.annotations.file_migrate).move_to_package = "envoy.extensions.tracers.fluentd.v4alpha";
```

Comment:
I prefer to use `v3` directly because our API version won't change in the future. The extension metadata could be used to indicate the extension is alpha.